tests/runtime/builtin

```diff
@@ -28,6 +28,11 @@ PROG i:ms:1 { printf("SUCCESS %d\n", elapsed); exit(); }
 EXPECT SUCCESS -?[0-9][0-9]*
 TIMEOUT 5
 
+NAME numaid
+PROG i:ms:1 { printf("SUCCESS %d\n", numaid); exit(); }
+EXPECT SUCCESS -?[0-9][0-9]*
```

Comment:
for a future fix it it would be nice if we could validate the numa node id this outputs, although that might be tricky with the ci (not sure how many nodes it has). Its to rule out that we don't accidentally return a 0. In the current form the doesn't catch anything the earlier tests wouldn't. But lets not block the MR on it

Reply:
Since there should be no leading zeros, could we do this?
```suggestion
EXPECT SUCCESS -?[1-9][0-9]*
```
That would rule out a 0.

Reply:
isn't 0 a valid numa node id?

Reply:
Ah, it is, so this wouldn't work.